### ISSUE TITLE
Replacement of gridelements

### ISSUE THREAD

**Issue by twixwix** [t3solution/t3sbootstrap#91]
Since I did the replacement of "gridelements" with the "Container" I get everywhere errors:

> ERROR: Content Element with uid "1493" and type "two_columns" has no rendering definition!

> ERROR: Content Element with uid "1491" and type "card_wrapper" has no rendering definition!

Is there a simple solution because I would really like to walk with you guys but otherwise have to go back to gridelements. Didn't saw any includes in the template about the "container" extension.

**Comment by t3solution** [comment]
You have to read the information carefully: https://www.t3sbootstrap.de/version-455
There is an Upgrade Wizard!

**Comment by twixwix** [comment]
yeah I saw that but in my case I did not have any content it is a fresh installation but when I add a "container" element then on all of them is coming this message.

**Comment by t3solution** [comment]
... your first sentence: Since I did the replacement of "gridelements" with the "Container" I get everywhere errors
Since when is content (here gridelements) in a fresh installation?
That confuses me now!

**Comment by twixwix** [comment]
So to explain the whole thing briefly. We have set up a new installation and imported the content from an existing site where some elements from the grid elements were used but there are only 1 or 2 so it would be easy to start directly on the new one. However, when I try to use the new containers, I get the above mentioned error message. To be on the safe side, I ran the tasks in the planner again, but that didn't change anything. During the installation we installed the gridelements at the beginning and removed them afterwards, could this be another reason for the message?
